node/src/hopper/routing_service.rs

```diff
@@ -420,11 +420,13 @@ impl RoutingService {
                 }
                 match self.routing_service_subs.to_accountant_routing.try_send(
                     ReportRoutingServiceProvidedMessage {
-                        timestamp: SystemTime::now(),
-                        paying_wallet: payer.wallet,
-                        payload_size,
-                        service_rate: self.per_routing_service,
-                        byte_rate: self.per_routing_byte,
+                        service: ServiceProvided {
```

Comment:
I dunno why you nested it like that, looks unnecessary, unless there's a sibling field.